web/sites/default/config/block.block.cloudy_views_block__assigned_groups_block_assigned_groups.yml

```diff
@@ -1,22 +1,22 @@
-uuid: a5d412b8-9413-4d80-9683-197b9133f0d0
+uuid: 787a2bb0-3681-473f-ad11-af01818f6e28
 langcode: en
 status: true
 dependencies:
   config:
-    - views.view.my_groups
+    - views.view.assigned_groups
   module:
     - system
     - views
   theme:
     - cloudy
-id: views_block__my_groups_block_1
+id: cloudy_views_block__assigned_groups_block_assigned_groups
 theme: cloudy
 region: content
 weight: -74
 provider: null
-plugin: 'views_block:my_groups-block_1'
```

Comment:
Should `block_1` be deleted from the `my_groups` view if it's no longer in use?

Reply:
The block is currently disabled. I didn't remove it completely in the hopes that the Drupal issue that's preventing us from using this display eventually gets resolved and then we can reenable it the remove the other redundant view.